app/lib/package/models.dart

```diff
@@ -148,6 +148,23 @@ class Package extends db.ExpandoModel {
   }
 
   bool isNewPackage() => created.difference(DateTime.now()).abs().inDays <= 30;
+
+  /// List of tags from the flags on the current [Package] entity.
+  List<String> getTags() {
+    return <String>[
+      if (isDiscontinued) PackageTags.isDiscontinued,
+      if (isNewPackage()) PackageTags.isNew,
```

Comment:
Let's add a TODO: for `is:not-advertized` and `is:dart-1`, `publisher:<publisherId>` and `uploader:<...>`.

With those in-place (later) we can remove a lot of the stuff we have in search.